app/javascript/src/components/OrganizationSetup/index.tsx

```
@@ -3,10 +3,11 @@ import React, { useState } from "react";
 import Steps from "rc-steps";
 import "rc-steps/assets/index.css";
 import { useNavigate } from "react-router-dom";
-
 import companiesApi from "apis/companies";
 import MiruLogoWatermark from "common/MiruLogoWatermark";
+
 import { Paths } from "constants/index";
```

ESLint

[error] `constants/index` import should occur before import of `react`

(import/order)